framework/doc/content/source/auxkernels/ElementQualityAux.md

```diff
@@ -0,0 +1,20 @@
+# ElementQualityAux
+
+!syntax description /AuxKernels/ElementQualityAux
+
+# Description
+
+Computes per-element quality metrics and puts the value in a field.  An example is shown below using the `SHAPE` metric on an unstructured mesh.  In this case "perfectly" shaped elements have a value of `1.0` and as the quality degrades so does the value of the shape metric.
+
+!media media/framework/auxkernels/element_quality.png style=width:60%
+
+
+The `SHAPE` metric is a good one to use for quads and tris.  You can see the list of all possible metrics below in the documentation for the `metric` parameter.
```

Comment:
The metric parameter below doesn't seem to show a list of possible metrics.

Reply:
Well - that's dumb.  @aeslaughter any idea why the MOOSE Enum is not showing the possible options here: http://mooseframework.inl.gov/docs/PRs/12135/site/source/auxkernels/ElementQualityAux.html

?